src/Transfers/ProductDataStatusTransfer.php

```diff
@@ -11,8 +11,7 @@
  * @property string $item_not_ready_reason
```

Comment:
This file has been automatically changed by the `dev_tools/generate` command of the `transfers.php` file.